packages/integration-karma/test/component/LightningElement.getBoundingClientRect/index.spec.js

```diff
@@ -34,6 +34,6 @@ it('should return the host element dimensions', () => {
     document.body.appendChild(elm);
 
     const result = elm.getComponentBoundingClientRect();
-    expect(result.width).toBe(10);
-    expect(result.height).toBe(10);
+    expect(Math.abs(result.width - 10)).toBeLessThan(0.0001);
+    expect(Math.abs(result.height - 10)).toBeLessThan(0.0001);
```

Comment:
The best matcher for this is [`toBeCloseTo`](https://jasmine.github.io/api/edge/matchers#toBeCloseTo). 

```suggestion
    expect(result.width). toBeCloseTo(10);
    expect(result.height). toBeCloseTo(10);
```

I would also recommend adding a comment here explaining why we went with this approach instead of `toBe`. I am certain that we will forget in the future why this was needed.